ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from client.models import Client
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from django.contrib.auth.models import User
+
+
+USER_NAME = 'meitar'
+PASSWORD = 'pass1234'
+VALID_PRODUCT_NAME = 'Banana'
+INVALID_PRODUCT_NAME = 'Eggplant'
+USER_NAME = 'meitar'
+VALID_SUPPLIER_PRODUCT_ID = '1'
+INVALID_SUPPLIER_PRODUCT_ID = '0'
+VALID_DELIVERY_LOCATION_ID = '3'
+INVALID_DELIVERY_LOCATION_ID = '9797'
+VALID_QUANTITY = '5'
+ORDER_PRODUCT_ID = '1'
```

Comment:
The use of module-level constants in tests was discussed on a few other PRs already. The bottom line is that those constants are not really constants and if they are global to the module then one test may modify them for other tests, so tests may start affecting each other -- which is something we should avoid.

- If you refer to the constant on a single test/fixture, then there's no need to have it globally defined. Either use its literal value or define it as a local variable.
- If you need to refer to it multiple times, then make it a fixture. This way it is redefined for each test that consumes the fixture, so you get the original value reassigned at the beginning of each such test.